index.html

```diff
@@ -1024,6 +1024,44 @@ <h2>Inheritance and Default ACLs</h2>
           </p>
         </blockquote>
       </section>
+      <section id="acl-representation">
+        <h2>ACL Representation and Interpretation</h2
+        <p>
+          ACL resources are represented as described in [[!SOLIDWEBAC]]
+          <a href="https://github.com/solid/web-access-control-spec#representation-format">Representation
+          Format</a>. Implementations MUST inspect the ACL RDF for authorizations. Authorizations are identified by
+          type definition triples of the form <code>authorization_N rdf:type acl:Authorization</code>, where
+          <code>authorization_N</code> is the URI of an authorization. Implementations MUST use only statements
+          associated with an authorization in the ACL RDF to determine access, they MUST NOT dereference the
+          URI of the authorization. The authorizations MUST be examined to see whether they grant the requested
```

Comment:
What about agentGroup?  The URI would have to be dereferenced then.

Reply:
Well spotted @dannylamb ,  this clearly needs to be addressed so I'll propose a fix